### PR TITLE
Expose HttpContinueDelegate from HttpWebRequest.

### PR DESCRIPTION
**Pull request by tijoytom**
Fixes https://github.com/dotnet/corefx/issues/12512
The underlying winhttp API do not expose http 100
continue , so implementing ContinueDelegate is not
possible now.Making it a NOP in netcore for compat.

### CODE REVIEW

**Comment by tijoytom** [comment]
@davidsh @CIPop @karelz

File: src/System.Net.Requests/ref/System.Net.Requests.cs [hunk 1]
```diff
@@ -174,6 +175,7 @@ public partial class HttpWebRequest : System.Net.WebRequest
         public string Host { get { throw null; } set { } }
         public override string ConnectionGroupName { get { throw null; } set { } }
         public System.Net.Security.RemoteCertificateValidationCallback ServerCertificateValidationCallback { get; set; }
+        public HttpContinueDelegate ContinueDelegate { get { throw null; }set { } }
```
Comment (tijoytom) [inline src/System.Net.Requests/ref/System.Net.Requests.cs]: Sorry , this's wrong .. let me update the PR

**Comment by davidsh** [comment]
Even though this property will be a "no-op", it still preserves the value passed in the property setter and returns it via the getter.

So, you should add some tests to verify that.

**Comment by tijoytom** [comment]
Test Innerloop Windows_NT Debug Build and Test

**Comment by tijoytom** [comment]
Unreleated test failure 

Assert.NotNull() Failure
at System.IO.Tests.PathTooLongExceptionInteropTests.From_HR() in D:\j\workspace\windows_nt_de---4526f5ff\src\System.Runtime\tests\System\IO\PathTooLongException.InteropTests.cs:line

File: src/System.Net.Requests/tests/HttpWebResponseTest_1.7.cs [hunk 2]
```diff
@@ -1,46 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```
Comment (stephentoub) [inline src/System.Net.Requests/tests/HttpWebResponseTest_1.7.cs]: Why did these tests get deleted?

**Pull request merged**